houston/queries.go

```diff
@@ -494,6 +494,14 @@ mutation UpdateDeployment($deploymentId: Uuid!, $payload: JSON!, $cloudRole: Str
 			defaultAirflowImageTag
 		}
 	}`
+	AppVersionRequest = `
+	query AppConfig {
+		appConfig {
+			version
+			baseDomain
+		}
+	}
+	`
 	AppConfigRequest = `
```

Comment:
Why dont we just update the response parameters here. Everything else is the same. The code seems redundant. What do you think?

Reply:
because we want to avoid:
```
Error: API error (400): {"errors":[{"message":"Cannot query field \"configureDagDeployment\" on type \"AppConfig\".","locations":[{"line":8,"column":4}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED"}},{"message":"Cannot query field \"nfsMountDagDeployment\" on type \"AppConfig\".","locations":[{"line":9,"column":4}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED"}}]}
```

Reply:
@kushalmalani if you astro-cli above platform you run query with non-existing types:
```
query AppConfig {
		appConfig {
			version
			baseDomain
			smtpConfigured
			manualReleaseNames
			configureDagDeployment
			nfsMountDagDeployment
		}
	}
```

Reply:
why dont we remove `configureDagDeployment` from the response parameters in that request. Would that fix the issue?

Reply:
@kushalmalani because problem in request not in response. Please suggest change, TBH i don't fully understand. Do you want concatenate strings instead of duplication?

Reply:
Verfiied we need the new request. Its throwing an error for nfsMountDagDeployment and configureDagDeployment